rpm/kubelet.spec

```diff
@@ -6,6 +6,20 @@
 %global RPM_RELEASE 1
 %global ARCH amd64
 
+# If you want to build rpms for a prerelease version, you should set this to
+# the version suffix after MAJOR.MINOR.PATCH. For instance, if you want to
+# build 1.8.0-beta.1, set:
+#
+# %global KUBE_MAJOR 1
+# %global KUBE_MINOR 8
+# %global KUBE_PATCH 0
+# %global KUBE_VERSION_SUFFIX -beta.1
+#
+# However, the rpm version and name will still only be MAJOR.MINOR.PATCH, so
```

Comment:
why keep the rpm version and name the same? this seems error-prone at first glance.